app/scripts/modules/core/src/pipeline/config/stages/deploy/deployExecutionDetails.controller.js

```diff
@@ -79,9 +82,17 @@ module.exports = angular.module('spinnaker.core.pipeline.stage.deploy.details.co
       $scope.provider = context.cloudProvider || context.providerType || 'aws';
 
       $scope.changeConfig = {
+        buildInfo: context.buildInfo || {},
         commits: $scope.stage.context.commits,
         jarDiffs: $scope.stage.context.jarDiffs
       };
+      const serverGroupName = context['deploy.server.groups'][context.source.region][0];
```

Comment:
This was the CD/Orca fix/feature to add the build information to a single server group retrieval (Clouddriver, https://github.com/spinnaker/clouddriver/pull/1718), and add the previous/current build numbers to the commit diff task (Orca, https://github.com/spinnaker/orca/pull/1429).

See the previous comment about the jenkins reference.